packages/block-library/src/table/save.js

```diff
@@ -65,7 +77,12 @@ export default function save( { attributes } ) {
 										scope={
 											tag === 'th' ? scope : undefined
 										}
-										colSpan={ colspan }
+										colSpan={ normalizeRowColSpan(
```

Comment:
Should we not make sure these are validated on save rather than normalising on display?

Reply:
Considering the possibility of clipboard data containing `colspan` and `rowspan` being pasted, I thought it should be normalized in both the edit component and the save method. What do you think?

Reply:
I think it's better to normalise on paste

Reply:
I added the `transform` when pasted, i.e. in the `type: 'raw'` case.
The process is a bit complicated, but it should work as expected.